README.md

```diff
@@ -131,6 +131,10 @@ Thanks, and happy caching!
 
 Double-click on `Package.swift` in the root of the repository to open the project in Xcode.
 
+## Default branch
+
+The default branch of this repository is `main`. Between the initial commit and [`151ab3f`](https://github.com/dfed/CacheAdvance/commit/151ab3f1d57531b9ae7a9219d0d8a33300704595), the default branch of this repository was `master`. See #46 for more details on why this change was made.
```

Comment:
I _think_ it might automatically link to the commit message. Not totally sure though.

Reply:
Let's try it!
```suggestion
The default branch of this repository is `main`. Between the initial commit and 151ab3f1d57531b9ae7a9219d0d8a33300704595, the default branch of this repository was `master`. See #46 for more details on why this change was made.
```

Reply:
nope. that only works in PRs apparently.